### ISSUE TITLE
Add support for .traceurrc.js files

### ISSUE THREAD

**Issue by arv** [google/traceur-compiler#2101]
It would be nice to support having a `.rc` file which would provide the options to use when compiling.

This would be used when compiling using the command line and it would look in the same directory as the binary was invoked in a search parent directories if not there.

The format for the rc file would be a js file with a default exported object:

```js
// .traceurrc.js
module.exports = {
  modules: 'commonjs',
  classes: 'parse',
  ...
};
```

**Comment by arv** [comment]
@johnjbarton

**Comment by johnjbarton** [comment]
Are you thinking this would be optional or it would be required and would define the defaults?

My main concern is the extra mystery if you get an unexpected error and don't know it was a side effect of an option set in a hidden file.   But of course hidden files is how we spell 'convenience' ;-)
